desktop/cmp/contextmenu/ContextMenu.ts

```diff
@@ -20,6 +20,12 @@ export type ContextMenuSpec = MenuItemLike[] | ((e: MouseEvent) => MenuItemLike[
 
 export interface ContextMenuProps extends HoistProps {
     menuItems: MenuItemLike[];
+    /**
+     * Optional event because this component is sometimes used as the content in popovers, to
+     * create menu buttons.  In those cases, it is not expected that the click event contains any
```

Comment:
> because this component is sometimes used as the content in popovers, to create menu buttons

Is this not *always* how it's used?  What's an example of the "contextual information" here, or when would you expect to find something in the first event `e`?  Just as a developer I'm left a little uncertain as to what I would do with one event or the other.